src/main/java/com/redhat/jenkins/plugins/ci/CIBuildTrigger.java

```diff
@@ -83,7 +85,7 @@
 	private List<ProviderData> providers;
 
 	public static final transient ConcurrentMap<String, List<CITriggerThread>> triggerInfo = new ConcurrentHashMap<>();
-	public static final transient ConcurrentMap<String, Object> locks = new ConcurrentHashMap<>();
+    public static final transient ConcurrentMap<String, Object> locks = new ConcurrentHashMap<>();
```

Comment:
Note static fields do not have to be declared static not transient (for XStream at least).

Reply:
Resolved in latest commit.